core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -874,20 +843,70 @@ object JsonAST {
     }
   }
 
+  sealed trait AppendContainer[T] {
```

Comment:
Not sure I like this name. Why not just name it `StringAppendable` or `UnifiedAppendable`? Either way, a scaladoc indicating why it's here would be handy.

Also wondering if it wouldn't make sense to make these value classes… Not sure we'd gain an advantage here since we treat them as their trait, though.

Actually, now that I think about it, is there a strong reason why we're using Scala's `StringBuilder` instead of Java's which implements `Appendable`?